test/cookbooks/osl-matrix-test/recipes/synapse-admin.rb

```diff
@@ -0,0 +1,3 @@
+osl_synapse_admin 'test_admin' do
+  home_server 'chat.example.org'
+end
```

Comment:
I recommend you add another resource which tests the array method as well. In theory, you should be able to do this in the same recipe/suite.